nservicebus/upgrades/asp-saga-pruning.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Azure Storage Persistence upgrade Version 6.2.4 to 6.2.5
+summary: Instructions on how to patch Azure Storage Persistence when orphan saga index records appear.
+reviewed: 2016-06-30
+tags:
+ - upgrade
+related:
+ - nservicebus/sagas
+ - nservicebus/azure-storage-persistence
+---
+
+
+## Summary
+
+This document explains how to upgrade and patch a system for [Azure Storage Persistence bug #74](https://github.com/Particular/NServiceBus.Persistence.AzureStorage/issues/74) using NServiceBus.Azure hotfix release 6.2.5.
+
+WARNING: When upgrading to NServiceBus.Persistence.AzureStorage Version 1 and above, the following upgrade will need to be performed prior to beginning any other upgrade steps.
```

Comment:
@dbelcham - Should we mention the saga de-deduplication patch here in this warning? 

Something like:
WARNING: When upgrading to NServiceBus.Persistence.AzureStorage Version 1 and above, ensure that you have applied the [saga de-duplication patch](/nservicebus/upgrades/asp-saga-deduplication.md) followed by this patch. 

Thoughts?